README.md

```diff
@@ -14,13 +14,13 @@
 <table align="center">
     <tr>
         <td>
-            <img src="./docs/grc_eg.png" height="300px" />
+            <img src="./docs/grc_eg.png" height="200px" />
```

Comment:
what about specifying the width? Something in github's CSS must be force-scaling things – normally in HTML, when you specify one dimension, the other gets chosen to keep the aspect ratio correct. Here, this isn't happening, so I think it would be prudent to specify both dimensions explicitly in order to not depend on the "external" style squashing the images as it sees fit on whatever screen resolution the user has.